runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/config/BindingConfig.java

```diff
@@ -36,13 +38,15 @@
 
     public BindingConfig(
         String vault,
+        String name,
```

Comment:
Suggest moving `entry` after `type` and `kind` in parameter order, as it is logically dependent on those.